git/project_git_branches.md

```diff
@@ -1,8 +1,70 @@
+
+In this project we will be forking TheOdinProject's `git_branches` repository and learning to create a branch to work in and trial new feature for your the `git_branches` web page. If you want to try adding an experimental feature to your project, branching allows you add and change code whilst not modifying the `master` branch. Should the feature be successful, it can be included in `master` branch with a merge. Should the feature be deemed unnecessary, the branch can be deleted and development can continue on the `master` branch without having to remove any code added for the feature branch.
+
+## Learning Outcomes
+
+* Fork an existing repository from GitHub and clone it to you local machine.
+* Create a new branch in the repository and add new features.
+* Switch between branches and view the different state of the project.
+* Merge the feature branch into `master` and see the changes from the feature branch included in `master`
 
 ## Your Task
-Describe the requirements for the project in detail.
+
+1. Go to [TheOdinProject's git_branches](https://github.com/TheOdinProject/git_branches) repository on GitHub. This repository is a simple web page `index.html` that has been made specifically for this lesson. Fork the repository by clicking the button shown below.
+  <a href="http://i.imgur.com/fcJTCJU.png"><img class="tutorial-img" src="http://i.imgur.com/fcJTCJU.png" title="Click Fork icon" /></a>
+2. After a short wait you should now have your own fork of the `git_branches` repository at `https://github.com/Your-GitHub-Username/git_branches`. When you fork on GitHub you can think of it as creating a branch of the project that you own and have rights to modify and write (`push`) to as you please without affecting the original.
+3. In terminal cd  into the folder where you would like to place your forked copy of the curriculum and clone it with this line `git clone https://github.com/Your-GitHub-Username/git_branches.git`
+4. Now if we `cd git_branches` and run `git branch -v` we should see this:
+  ```language-bash
+   * master                f9230de Add index.html and describe project README.md
+
+  ```
+5. Let's create a new branch on the project called `new-feature` by entering `git branch new-feature`. Once done we should see a new branch in the list like so:
+  ```language-bash
+   * master                f9230de Add index.html and describe project README.md
+     new-feature           f9230de Add index.html and describe project README.md
+
+  ```
+6. Now you can see the original branch `master` and the new branch `new-feature`. You'll see the head on both is currently pointing to the commit `f9230de` and we are on branch `master` as signified by the `*`. To swap, use `git checkout branch-name`. In this case it will be `git checkout new-feature`. As you will often want to create a branch and then checkout to the new branch, git provides a shortcut `git checkout -b new-feature` to do these two commands in one line.
```

Comment:
I think this will read a little better split into 2 paragraphs

> 6. Now you can see the original branch `master` and the new branch `new-feature`. You'll see the head on  both is currently pointing to the commit `f9230de` and we are on branch `master` as signified by the `*`.
To swap, use `git checkout branch-name`. In this case it will be `git checkout new-feature`. As you will often want to create a branch and then checkout to the new branch, git provides a shortcut `git checkout -b new-feature` to do these two commands in one line.